src/config/docs.yaml

```
@@ -108,9 +108,13 @@ domains:
         chapters:
           - title: "Getting Started"
             pages:
-              - title: "ProteinPaint"
-                path: /docs/visualization-community/proteinpaint/
-              - title: "Frequently Asked Questions"
+              - title: "What is Visualizaton Community"
+                path: /docs/visualization-community/
+              - title: "What is ProteinPaint and GenomePaint"
```

```suggestion
              - title: "What are ProteinPaint and GenomePaint"
```

Should go what are? It's a tough call because sometimes they are considered the same and sometimes they are considered different. I'd say different here.